projects/1.20/assets/spit-splat/splat/lang/zh_cn.json

```
@@ -0,0 +1,26 @@
+{
+  "effect.splat.bubble_trap": "泡牢",
+  "entity.splat.bubble_entity": "泡牢",
+  "entity.splat.spit_splat": "唾溅兽",
+  "entity.splat.water_shockwave": "浪波",
+  "gui.splat.spit_splat_turret_on_off.label_is_now_following_you": "唾溅兽现在跟着你",
+  "item_group.splat.splat": "Splat",
+  "item.minecraft.lingering_potion.effect.bubble_trap_potion": "滞留型泡牢药水",
+  "item.minecraft.potion.effect.bubble_trap_potion": "泡牢药水",
+  "item.minecraft.splash_potion.effect.bubble_trap_potion": "喷溅型泡牢药水",
+  "item.minecraft.tipped_arrow.effect.bubble_trap_potion": "泡牢之箭",
+  "item.splat.bubble": "Bubble",
+  "item.splat.bubble_entity_spawn_egg": "泡牢刷怪蛋",
+  "item.splat.cooked_spit_splat": "熟唾溅兽",
+  "item.splat.raw_spit_splat": "生唾溅兽",
+  "item.splat.spit_splat_spawn_egg": "唾溅兽刷怪蛋",
+  "item.splat.splat_bones": "唾溅兽's 骨",
+  "item.splat.splat_scale": "唾溅兽鳞片",
+  "item.splat.splat_totem": "泼溅图腾",
+  "item.splat.splatter": "泼溅者",
+  "item.splat.splatter_ranged": "泼溅弹",
+  "item.splat.water_shockwave_spawn_egg": "浪波刷怪蛋",
+  "subtitles.bubblepop": "泡牢破裂",
+  "subtitles.splattershoot": "泼溅者射击",
+  "subtitles.water_shockwave": "浪波"
```

```suggestion
  "subtitles.water_shockwave": "浪波：泼溅"
```
具体以游戏内内容为准，我没开游戏，不清楚是什么声音